src/fees/FeeCalculatorBase.sol

```diff
@@ -31,10 +44,16 @@ abstract contract FeeCalculatorBase {
         return amounts;
     }
 
+    /// @notice Calculate the amount with the fee included for the given amount
+    /// @param amount The amount to calculate the total with the fee included
```

Comment:
Quite confusing on those fee calculating functions. Can we just simply say "The amount to be calculated"? All of those `amount` should be the amount without fee being included right?

Reply:
Agreed. Let's use with the comment style as you've suggested.

Reply:
Removed the previous comment.

The `@param amount` in both `calculateFee` functions are fee included. Do you think `calculateAmountWithFee` should remove those `included` terms?